docs/source/algorithms.rst

```diff
@@ -184,6 +184,73 @@ processor:
 
 When a `primitive` is not available on a processor, a conversion occurs automatically after the computation is complete.
 
+Batch jobs
+++++++++++
+
+The ``Sampler`` can setup a batch of different sampling tasks within a single job. Such a job enables you to gain
+some time (overhead of job management) as well as tidy up your job list, especially when running on the Quandela Cloud
+(but it can still be used in a local simulation context).
+
+The system relies on defining a circuit containing variable parameters, then with each iteration of the batch job,
+different values of these parameters can be set, along the input state and the `detected photons filter`. Each iteration
+must define a value for all variable parameters.
+
+>>> c = BS() // PS(phi=pcvl.P("my_phase")) // BS()
+>>> processor = pcvl.RemoteProcessor("qpu:ascella", token_qcloud)
+>>> processor.set_circuit(c)
+>>> sampler = Sampler(processor)
+>>> sampler.add_iteration(circuit_params={"my_phase": 0.1},
+>>>                       input_state=BasicState([1, 1]),
+>>>                       min_detected_photons=1)  # You can add a single iteration
+>>> sampler.add_iteration_list([
+>>>     {"circuit_params": {"my_phase": i/2}, "input_state": BasicState([1, 1]), "min_detected_photons": 1}
```

Comment:
J'utiliserais le même formatage de method qu'au dessus afin que l'utilisateur puisse comprendre la correspondance au premier coup d'oeil donc un truc comme ça :
sampler.add_iteration_list([
    {"circuit_params": {"my_phase": i/2}, 
    "input_state": BasicState([1, 1]), 
    "min_detected_photons": 1}
        for i in range(1, 6)])

Reply:
Done